src/containers/PatientDetails/PatientDocument/usePatientDocument.ts

```diff
@@ -144,10 +144,14 @@ export function usePatientDocument(props: Props): {
         }
 
         if (isSuccess(provenanceResponse)) {
-            const provenance = provenanceResponse.data[0];
+            const descSortedProvenances = [...provenanceResponse.data].sort((a, b) =>
+                b.recorded.localeCompare(a.recorded),
+            );
```

Comment:
I guess it can be sorted just by `[...provenanceResponse.data].sort()` as localeCompare anyways compare dates as strings

Reply:
[...provenanceResponse.data].sort((a) => [a.recorded, b.recorded].sort())